demo/DemoGitHubUtils.php

```diff
@@ -0,0 +1,29 @@
+<?hh // strict
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+namespace Facebook\ShipIt;
+
+final class DemoGitHubUtils extends ShipItGitHubUtils {
+
+  static string $committer_name = "FBShipIt Demo Committer";
+  static string $committer_user = 'CHANGEME';
+  static string $committer_email = "demo@example.com";
```

Comment:
Sharing feedback from internal linting

```suggestion
  static string $committerName = "FBShipIt Demo Committer";
  static string $committerUser = 'CHANGEME';
  static string $committerEmail = "demo@example.com";
```